lib/ui/main.dart

```diff
@@ -7,9 +7,11 @@ import 'package:flutter/material.dart';
 import 'package:flutter_slidable/flutter_slidable.dart';
 import 'package:font_awesome_flutter/font_awesome_flutter.dart';
 import 'package:alert_me/domain/mapper/TimeUtil.dart';
+import 'package:timezone/timezone.dart' as tz;
 
 void main() {
   NotificationService().initNotification();
+  tz.initializeDatabase([]);
```

Comment:
is tz being used anythwhere?

Reply:
Yes. This line is needed otherwise we would have exception `LateInitializationError: Field '_local' has not been initialized.`